memor/params.py

```diff
@@ -86,9 +86,56 @@ class LLMModel(Enum):
     CLAUDE_2 = "claude-2"
     CLAUDE_INSTANT = "claude-instant"
 
-    LLAMA3_70B = "llama3-70b"
-    LLAMA3_8B = "llama3-8b"
+    LLAMA_2_7B = "llama-2-7b"
+    LLAMA_2_13B = "llama-2-13b"
+    LLAMA_2_70B = "llama-2-70b"
+    LLAMA_3_8B = "llama-3-8b"
+    LLAMA_3_8B_INSTRUCT = "llama-3-8b-instruct"
+    LLAMA_3_70B = "llama-3-70b"
+    LLAMA_3_70B_INSTRUCT = "llama-3-70b-instruct"
+    LLAMA_3_1_8B = "llama-3.1-8b"
+    LLAMA_3_1_8B_INSTRUCT = "llama-3.1-8b-instruct"
+    LLAMA_3_1_70B = "llama-3.1-70b"
+    LLAMA_3_1_70B_INSTRUCT = "llama-3.1-70b-instruct"
+    LLAMA_3_1_405B = "llama-3.1-405b"
+    LLAMA_3_1_405B_INSTRUCT = "llama-3.1-405b-instruct"
+    LLAMA_3_2_1B = "llama-3.2-1b"
+    LLAMA_3_2_1B_INSTRUCT = "llama-3.2-1b-instruct"
+    LLAMA_3_2_3B = "llama-3.2-3b"
+    LLAMA_3_2_3B_INSTRUCT = "llama-3.2-3b-instruct"
+    LLAMA_3_2_11B = "llama-3.2-11b"
+    LLAMA_3_2_11B_INSTRUCT = "llama-3.2-11b-instruct"
```

Comment:
Please check this model once again. I believe it should be defined as `LLAMA_3_2_11B_VISION = "llama-3.2-11b-vision"`.

https://huggingface.co/collections/meta-llama/llama-32-66f448ffc8c32f949b04c8cf